src/renderer/hub.css

```
@@ -0,0 +1,184 @@
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: transparent;
+  overflow: hidden;
+  font-family: 'Rajdhani', 'Segoe UI', system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Rajdhani" (font-family-name-quotes)

(font-family-name-quotes)